internal/installation/paths.go

```diff
@@ -38,6 +38,10 @@ func InstallRoot(path string) (string, error) {
 		return "", errs.Wrap(err, "Could not find install marker file in path")
 	}
 
+	if !isValidInstallPath(filepath.Dir(installFile)) {
```

Comment:
Will this affect our local `build/` roots? i.e. will our local build directories considered to be invalid with this change?

Reply:
No, we will hit the code that facilitates that use case before validating the install path.